### PR TITLE
Add Vienna music venues

### PR DESCRIPTION
**Pull request by sotojuan**
Just adds some Vienna music venues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@
     - [Salzburg](#salzburg)
     - [Innsbruck](#innsbruck)
     - [Graz](#graz-1)
+- [Music](#museums)
```
Comment (mxstbr) [inline README.md]: I doubt this is `#museums` 😉 let's also add Vienna as a subcategory here.

**Review by mxstbr — CHANGES_REQUESTED** [review]
Awesome! One tiny nitpick then we can ship this 🔥

**Comment by sotojuan** [comment]
Whoops. Should be fine now!

**Review by mxstbr — APPROVED** [review]

**Comment by mxstbr** [comment]
Awesome thanks!

**Pull request merged**